vcr/verifier/verifier.go

```diff
@@ -135,6 +126,14 @@ func (v verifier) Verify(credentialToVerify vc.VerifiableCredential, allowUntrus
 		return err
 	}
 
+	revoked, err := v.IsRevoked(*credentialToVerify.ID)
+	if revoked {
+		return errors.New("credential is revoked")
+	}
+	if err != nil {
```

Comment:
as a matter of principle, better to check `err` first, then the regular return values?